public/app/features/browse-dashboards/components/NameCell.tsx

```diff
@@ -93,7 +93,7 @@ export function NameCell({ row: { original: data }, onFolderClick, treeID }: Nam
               onClick={() => {
                 reportInteraction('manage_dashboards_result_clicked');
               }}
-              href={item.url}
+              href={getFolderURL(item.url)}
```

Comment:
I'm not sure if we construct this URL elsewhere after recieving the response back? It might be better to do this there? (im trying to refresh my memory of the code....)

Reply:
https://github.com/grafana/grafana/blob/14446f4e9b961df25af88a629c98ceaaa237af96/public/app/features/browse-dashboards/api/services.ts